src/main/resources/migration/common/V42__add_postponement_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE postponement_reason
+(
+    id SERIAL NOT NULL constraint postponement_reason_pk PRIMARY KEY,
+    reason_type varchar(60) NOT NULL,
+    assessment_id INTEGER NOT NULL,
+    created_timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    constraint fk_postponement_reason_assessment foreign key (assessment_id) references assessment(id)
+);
```

Comment:
meh.. missing newline character from end of file

Reply:
will sort